### PR TITLE
git-hooks-go 1.3.0 (new formula)

### PR DESCRIPTION
**Pull request by SeekingMeaning**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

I'm not sure what edits I should make to `Formula/git-hooks.rb`

### CODE REVIEW

File: Formula/git-hooks-go.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  desc "Git hooks manager"
+  homepage "https://git-hooks.github.io/git-hooks"
+  url "https://github.com/git-hooks/git-hooks/archive/v1.3.0.tar.gz"
+  sha256 "518eadf3229d9db16d603290634af8ae66461ec021edf646e8bca49deee81850"
+  license "MIT"
+  head "https://github.com/git-hooks/git-hooks.git"
+
+  depends_on "go" => :build
+
+  conflicts_with "git-hooks", :because => "both install `git-hooks` binaries"
```
Comment (chenrui333) [inline Formula/git-hooks-go.rb:11]: As `git-hooks` deprecated, why dont we just rewrite the original formula with the golang version?

Comment (chenrui333) [reply Formula/git-hooks-go.rb:11]: Assuming the golang edition is the same functionality as the old one.

**Comment by chenrui333** [comment]
relates to #56879

File: Formula/git-hooks-go.rb (hunk 1 above)
Comment (SeekingMeaning) [reply Formula/git-hooks-go.rb:11]: Yeah I don't see why not, how would we go about doing that? (how many commits, what to put for commit messages, etc)

Comment (SeekingMeaning) [reply Formula/git-hooks-go.rb:11]: No the functionality is different

Comment (SMillerDev) [reply Formula/git-hooks-go.rb:11]: Yeah, I had this conversation with Samford before. The two are different so we can't overwrite them.

**Review by SMillerDev — APPROVED** [review]
Thanks @SeekingMeaning  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

File: Formula/git-hooks-go.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/git-hooks-go.rb:11]: I see, sgtm.

**Review by chenrui333 — APPROVED** [review]

**Pull request closed without merging**